app/controllers/journeys_controller.rb

```diff
@@ -1,10 +1,14 @@
 class JourneysController < ApplicationController
+  include NotificationSubscriptionsHelper
+
   def show
     @journey = Journey.published.find_by!(slug: params[:id])
     @next_step = @journey.steps.order(:position).first
 
     load_newest_user_journey(current_user, @journey)
 
+    @journey.description = @journey.description.gsub(/(?<=<insert_form>).*?(?=<\/insert_form>)/) {|s| render_notification_subscription_component_to_string([s])} if @journey.description.include? '<insert_form>'
```

Comment:
insert_form je prilis genericke. Normalne nech to je `<notification-subscription types="typ1,typ2">` chceme aj podporu pre viac checkboxov/typov - tu sa to podla mna neda.